app/javascript/utilities/hooks/useGlobalKeyEventListener.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'preact/hooks';
+
+const FOCUSED_TAG_EXCLUDE_LIST = ['INPUT', 'TEXTAREA'];
+
+export const registerGlobalKeyEventListener = (keys, callback) => {
```

Comment:
syntax (non-blocking): There is no need for these to be arrow functions (here and below), they can be regular functions.

API documentation (blocking): The function requires documentation as it is being exported. This provides documentation in the file directly, but the JSDoc can also provide Intellisense information to a developer, e.g. VS Code. For an example of JSDoc comments in action in the project, see https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js